src/AmazonPHP/SellingPartner/ObjectSerializer.php

```diff
@@ -398,6 +394,24 @@ public static function deserialize(Configuration $configuration, $data, string $
         return $instance;
     }
 
+    /**
+     * Define a class name to disable enum value validation for incomplete model definitions.
+     *
+     * Due to an incomplete Amazon model definition, an unknown enum value in the API response would result in an exception and error during validation.
+     * This array defines in advance the class name of the enum value that will invalidate the validation, and returns it to the caller.
+     * https://github.com/amazon-php/sp-api-sdk/issues/191
+     * https://github.com/amazon-php/sp-api-sdk/issues/156
+     *
+     * @return array enum value class name
+     */
+    private function getBrokenModelDefinitions() : array
```

Comment:
I fixed this directly at 4.x branch, `static` definition was missing.

Reply:
@norberttech 
Ahhh, sorry for the oversight.
Thanks again for the correction.